plugins/window-rules/view-action-interface.cpp

```diff
@@ -206,6 +237,40 @@ void view_action_interface_t::_unminimize()
     _view->set_minimized(false);
 }
 
+void view_action_interface_t::_make_sticky()
+{
+    _view->set_sticky(1);
+}
+
+void view_action_interface_t::_always_on_top()
```

Comment:
We are really duplicating code here. Doesn't wm-actions have a signal to toggle always-on-top for a view?

Reply:
The code is not really duplicated. In this case I thought it would make more sense that `set always_on_top` meant enable it, and not toggle it (and the function has been modified to reflect that). Consider if you tied it to maximize. Every *other* time you maximize the view, it would become always on top and the other times it would be normal floating. So yes, I could use the wm_actions_toggle_above signal, but I think it wouldn't do what the rule says.

Reply:
> I see. Then it would make sense to either extend the wm-actions signal or add a new one. Duplicating functionality is not a good idea in any case.

Sure. I will find a better way.